### PR TITLE
i15 // Spanin Tools

### PR DESCRIPTION
**Pull request by curtisim0**
[#15]

### CODE REVIEW

**Review by mali0171 — COMMENTED** [review]
Looks good. Regex is changed in both .py and .xml

File: cpt_putative_isp/cpt.py
Comment (mali0171) [inline cpt_putative_isp/cpt.py:1]: Regex changed from built-in re in Python. Looks good.

File: cpt_putative_isp/generate-putative-isp.xml
Comment (mali0171) [inline cpt_putative_isp/generate-putative-isp.xml:1]: Regex version changed in xml too. Looks good.

Comment (mali0171) [inline cpt_putative_isp/generate-putative-isp.xml:1]: Should these versions be updated in the main macros XML file as well? Or only needed for this tool?

File: cpt_putative_osp/generate-putative-osp.py
Comment (mali0171) [inline cpt_putative_osp/generate-putative-osp.py:1]: Good job removing unnecessary calls.

File: cpt_find_spanins/spaninFuncs.py
Comment (mali0171) [inline cpt_find_spanins/spaninFuncs.py:1]: Formatting or linter change for the TypeError?

File: cpt_find_spanins/cpt-macros.xml
Comment (mali0171) [inline cpt_find_spanins/cpt-macros.xml:1]: Why is all of this being removed?

File: cpt_find_spanins/macros.xml
Comment (mali0171) [inline cpt_find_spanins/macros.xml:1]: What does this addition do?

File: cpt_find_spanins/findSpanin.py
Comment (mali0171) [inline cpt_find_spanins/findSpanin.py:1]: Good job noticing the change!

**Review by mali0171 — COMMENTED** [review]
Overall looks good. I left comments on most of the commits. I had some questions that I commented, most of them are pretty much me trying to understand why certain things were done or what was the purpose of changing certain things.

File: cpt_find_spanins/macros.xml
Comment (curtisim0) [reply cpt_find_spanins/macros.xml:1]: symbolic link to just point to the same one.

File: cpt_find_spanins/cpt-macros.xml
Comment (curtisim0) [reply cpt_find_spanins/cpt-macros.xml:1]: not used

File: cpt_find_spanins/spaninFuncs.py
Comment (curtisim0) [reply cpt_find_spanins/spaninFuncs.py:1]: format, it was going past the 88 char limit

**Review by mali0171 — APPROVED** [review]
Overall looks good. Try pushing to Galaxy and see if this tool works now.

**Pull request merged**